test/routes/changePassword.test.js

```diff
@@ -0,0 +1,31 @@
+const test = require('tape');
+const supertest = require('supertest');
+const app = require('../../server/app');
+const dbBuild = require('../../server/database/config/dbbuild');
+
+test('Test /users/change-password route', (t) => {
+  dbBuild()
+    .then(() => {
+      supertest(app)
+        .put('/api/v1/users/change-password')
+        .send({
+          userId: 1,
+          newPassword: 'mai',
+          oldPassword: 'Fatma123',
```

Comment:
Yes, the test will pass but have you checked your local database and see the value of the new password?

Reply:
this is kind of a duplicate pr where we fixed this one already me and fatma